docs/samples/docker/litellm.md

```diff
@@ -0,0 +1,234 @@
+# LiteLLM (OpenAI Proxy) with Presidio
+
+LiteLLM supports [Microsoft Presidio](https://github.com/microsoft/presidio/) for PII masking. 
+
+[👉 **Refer to LiteLLM Docs for detailed guide**](https://docs.litellm.ai/docs/proxy/pii_masking)
```

Comment:
I would make it a bit more explicit that the below instructions are based a basic quick start running (f.e. LiteLLM running as docker on port 4000 is not mentioned here or on the pii masking quickstart)